### PR TITLE
Add new mode to see not bound type

### LINKED ISSUE #461: List the ignored types
So that it will be easier to see which one are missing in the `Gir.toml` file.

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #461.

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
PRs on GTK are following. :)

File: src/library.rs [hunk 1]
```diff
@@ -720,6 +743,33 @@ impl Library {
         library
     }
 
+    pub fn show_non_bound_types(&self, env: &Env) {
+        let not_allowed_ending = ["Class", "Private", "Func", "Callback", "Accessible", "Iface",
+                                  "Type"];
+        let namespace_name = self.namespaces[MAIN_NAMESPACE as usize].name.clone();
+        for x in self.namespaces[MAIN_NAMESPACE as usize].types.iter() {
```
Comment (EPashkin) [inline src/library.rs]: Maybe use `namespace(MAIN_NAMESPACE)` ?

File: src/library.rs [hunk 2]
```diff
@@ -720,6 +743,33 @@ impl Library {
...
+            if let Some(ref x) = *x {
+                let name = x.get_name();
+                if !not_allowed_ending.iter().any(|s| name.ends_with(s)) {
+                    let full_name = format!("{}.{}", namespace_name, name);
+                    let version = x.get_deprecated_version();
+                    let depr_version = version.clone()
+                                              .unwrap_or(env.config.min_cfg_version.clone());
```
Comment (EPashkin) [inline src/library.rs]: Version is `Copy`, so IMHO .clone unneeded.

File: src/config/gobjects.rs [hunk 3]
```diff
@@ -1,4 +1,3 @@
-use std::ascii::AsciiExt;
```
Comment (EPashkin) [inline src/config/gobjects.rs]: This still needed in stable :(

**Comment by EPashkin** [comment]
README.md IMHO need update section "`gir` Modes"

**Comment by EPashkin** [comment]
@GuillaumeGomez Thanks, merge when you see that it ready.

File: src/config/gobjects.rs (hunk 3 above)
Comment (GuillaumeGomez) [reply src/config/gobjects.rs]: Ah damn.

**Comment by GuillaumeGomez** [comment]
Updated and added it in the README.

**Comment by GuillaumeGomez** [comment]
I merge then.

**Pull request merged**